main.md

```diff
@@ -275,6 +275,121 @@ claim values), SD-JWT Salt/Value Containers (containing the mapping of the
 plain-text claim values and the salt values), and SD-JWT Releases (containing a
 subset of the same mapping).
 
+## The Challenge of Canonicalization
```

Comment:
should this be in Implementation considerations section..?

Reply:
Not sure - readers might miss it there.